admin/app/main/views/dashboard.py

```diff
@@ -402,8 +402,8 @@ def get_months_for_financial_year(year, time_format='%B'):
     return [
         month.strftime(time_format)
         for month in (
-            get_months_for_year(4, 13, year) +
-            get_months_for_year(1, 4, year + 1)
+            get_months_for_year(7, 13, year) +
+            get_months_for_year(1, 7, year + 1)
```

Comment:
I think the financial year for say 2018 would be 2017-07-01 to 2018-06-30 not 2018-07-01 to 2019-06-30.

https://en.wikipedia.org/wiki/Fiscal_year#Australia
https://www.taxback.com/blog/australia-key-tax-dates/

Reply:
You're right that 17/18 financial year has "year=2018". Does this affect this section of code specifically or the PR?